git/cmd.py

```diff
@@ -13,6 +13,8 @@
 import errno
 import mmap
 
+from collections import OrderedDict
```

Comment:
Hi @boegel,

Indeed, with release 2.0 we officially dropped support for py2.6, assuming that this move is fine as py2.6 is not maintained anymore.

Nonetheless, I see no harm in using the alternate odict implementation, and would very much welcome a PR.

Thank you!